### ISSUE TITLE
Missing attributes and incorrect keyboard controls on tab control

### ISSUE THREAD

**Issue by Decrepidos** [govau/design-system-site#447]
Hi. 
Great to see a Government department dedicated to inclusion.

I found two issues with the tab control. You are missing aria-selected on the selected tab. A tab control should be one tab stop. arrow keys should move focus between the tabs.
WAI-ARIA Authoring Practices 1.1 3.22 Tabs
https://www.w3.org/TR/wai-aria-practices/#tabpanel

I also highly recommend not wrapping the tab control in a navigation landmark. This is not navigation. To identify the purpose of the tab control to screen reader users a better approach is to include an aria-label on the role tablist.

The tab control can be found on the Accordions example page and has the tab heading HTML and REACT.
https://designsystem.gov.au/components/accordion/

As an aside the accordion trigger used in the side menu should be a button or at the very least the < a > should have role of button. Code pasted below. This is announced by screen readers as 'link expanded', which may cause confusion.
`<a href="#accordion-Released" class="au-accordion__title js-accordion au-accordion--open" aria-controls="accordion-Released" aria-expanded="true">Released <span class="badge badge--Released">26</span></a>`

Regards
Laurence

**Comment by Decrepidos** [comment]
@sukhrajghuman you’re welcome 😬.
